calculator-app/src/App.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import './App.css';
+import Button from './components/Button';
+import Input from './components/Input'
+import ClearButton from './components/ClearButton'
+import * as math from 'mathjs'
+
+class App extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      input: ""
+    }
+  }
+
+  updateInput = (val) => {
+    this.setState({ 
+      input: this.state.input + val});
+  }
+
+  handleClear = () => {
+    this.setState({
+      input: ""
+    })
+  }
+
+  handleEqual = () => {
+    this.setState({ input: math.evaluate(this.state.input)})
+  }
+
+  handleClick = (val) => {
+    this.updateInput(val)
+  }
+
+  render() {
+    return (
+      <div className="App">
+        <div className="calc-wrapper">
+          <Input input={this.state.input}></Input>
+          <div className="row">
+            <Button handleClick={this.updateInput}>7</Button>
```

Comment:
Since you keep using the Button component, you could have an array of ["+", "=", "%" ... etc] and loop through it and for each of the elements render the Button component. To save you some hardcoding :)